.ruby-version

```diff
@@ -0,0 +1 @@
+2.6.5
```

Comment:
@cmharlow I'm confused about this here as I see cob_datapipeline as mainly a python project.  I thought .ruby-versions would go into the respective ruby projects.

Reply:
this runs ruby vis-a-vis the traject libraries.

Reply:
But, this wouldn't ever get used because when we call traject in this project that happens inside of a temporary directory where this file wont exist.  The reason it works for the cob_index, cob_az_index etc is that we clone into that tmp directory then cd into the cloned directory before running the script.

Reply:
yes, i know all that. just closing instead.